ports/stm/boards/stm32f411ce_blackpill/pins.c

```diff
@@ -35,5 +35,8 @@ STATIC const mp_rom_map_elem_t board_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_C14), MP_ROM_PTR(&pin_PC14) },
     { MP_ROM_QSTR(MP_QSTR_C13), MP_ROM_PTR(&pin_PC13) },
     { MP_ROM_QSTR(MP_QSTR_LED), MP_ROM_PTR(&pin_PC13) },
+
+    { MP_ROM_QSTR(MP_QSTR_SDA), MP_ROM_PTR(&pin_PB07) },
+    { MP_ROM_QSTR(MP_QSTR_SCL), MP_ROM_PTR(&pin_PB06) },
```

Comment:
Did you mean to add these pins in this PR? There is no `board.I2C()` defined?

Reply:
1. The SDA/SCL pins commit looks like it got added to the pull request after I'd created it when I added those to my fork - wasn't the intention that they be part of this pull
2. Yes, tested W25Q64FV working with build of 6.1.0-beta.2 and head.

Reply:
could you push a new commit that removes the SDA and SCL pins?

Reply:
Ok - will try that